ragna/assistants/_anthropic.py

```diff
@@ -36,12 +36,50 @@ def _instructize_system_prompt(self, sources: list[Source]) -> str:
             + "</documents>"
         )
 
-    async def answer(
-        self, messages: list[Message], *, max_new_tokens: int = 256
+    def _render_prompt(self, prompt: Union[str, list[Message]]) -> Union[str, list]:
```

Comment:
How can this return a `str`?

Reply:
yep, good call - I had originally meant to try to make all assistants follow the same in/out pattern, but I see I didn't even keep it in exl2 where I just set `-> str` - will drop the Union and make it just `list`